### ISSUE TITLE
CouldNotRetrieveTranscript

### ISSUE THREAD

**Issue by cantwbr** [jdepoix/youtube-transcript-api#25]
`YouTubeTranscriptApi.CouldNotRetrieveTranscript(video_id)` has been raised for the video: https://www.youtube.com/watch?v=pFQzqK3MrTc

```
from youtube_transcript_api import YouTubeTranscriptApi
print(YouTubeTranscriptApi.get_transcript("pFQzqK3MrTc"))
```

Subtitles and the video itself are available.

**Comment by jdepoix** [comment]
Apparently Google just changed something about how the their Transcript API works, therefore this module is currently not working at all. I am looking in to this and trying to fix it asap!

**Comment by jdepoix** [comment]
Just fixed this with v0.1.8. It is now available on PyPi. Sorry for the inconveniences!
